src/Uno.UI/UI/Xaml/Media/Imaging/SvgImageSource.Android.cs

```diff
@@ -67,6 +57,15 @@ private async Task<ImageData> GetSvgImageDataAsync(CancellationToken ct)
 					return await ReadFromStreamAsync(stream, ct);
 				}
 
+				if (AbsoluteUri.IsLocalResource())
```

Comment:
Isn't a similar change applicable to normal bitmaps/analogous code for other platforms? Maybe not, but rather checking

Reply:
Thinking about it, it may not be the right way to do it, since reading getting the asset ID is going to lead to faster loading. I'll rework that and adjust images as needed.

Reply:
Actually, this may be related to https://github.com/unoplatform/uno/issues/12475, we will fix this at the same time.